pyriemann/utils/distance.py

```diff
@@ -323,6 +323,65 @@ def distance_logeuclid(A, B, squared=False):
     return distance_euclid(logm(A), logm(B), squared=squared)
 
 
+def distance_poweuclid(A, B, p, squared=False):
+    r"""Power Euclidean distance between SPD/HPD matrices.
+
+    The power Euclidean distance of order :math:`p` between two SPD/HPD
+    matrices :math:`\mathbf{A}` and :math:`\mathbf{B}` is [1]_:
+
+    .. math::
+        d(\mathbf{A},\mathbf{B}) = \frac{1}{|p|}
+        \Vert \mathbf{A}^p - \mathbf{B}^p \Vert_F
```

Comment:
What is F in this case? The vector space?

Reply:
F for Frobenius norm.